saga-demo/saga-k8s-resources/spring-demo/booking.yaml

```diff
@@ -45,7 +45,7 @@ spec:
     spec:
       containers:
       - name: booking
-        image: booking:0.3.0-SNAPSHOT
+        image: swr.cn-north-1.myhuaweicloud.com/saga/booking:0.3.0-SNAPSHOT
```

Comment:
Can we change the url from the outside?
User may not use the swr service like we use.